README.md

```diff
@@ -1,2 +1,3 @@
 # work-experience
 Test repository for new starters to fork and make changes
+anil 24/08/2021
```

Comment:
Can you now modify this README to contain a little more information. Correctly formatted I would like...
First name 
Short bio
a numbered list of 3 things you would like to learn on your work experience
a small code block in python where you
define your first name in a variable,
define your age in a variable ,
print "Hello World! My name is {YourName} and I am {YourAge} years old"

Reply:
see: https://www.markdownguide.org/cheat-sheet/